src/containers/BooksForm.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+const categories = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+
+export default function BooksForm() {
```

Comment:
## Convert to ES6 syntax
- [ ] We need to ensure we are using ES6 syntax all throughout the codebase. Kindly convert this `BooksForm` component function to an arrow based function.
```suggestion
export default const BooksForm = () => {
```